app/__init__.py

```diff
@@ -0,0 +1,42 @@
+#!flask/bin/python3/
+from flask import Flask, Blueprint
+from flask_restful import Api
+from flask_sqlalchemy import SQLAlchemy
+
+from config import config
+
+db = SQLAlchemy()
+
+
+def create_app(config_name):
+    # Initialization of the application after runtime.
+
+    app = Flask(__name__)
+    app.config.from_object(config[config_name])
+    config[config_name].init_app(app)
+    db.init_app(app)
+    api_blueprint = Blueprint('api', __name__, url_prefix='/api/v1')
+    api = Api(api_blueprint)
+
+    # resources
+    from app.resources.abucketlist import ABucketList
+    from app.resources.abucketlistitem import ABucketListItem
+    from app.resources.allbucketlistitems import AllBucketListItems
+    from app.resources.allbucketlists import AllBucketLists
+    from app.auth.register_user import Register_User
```

Comment:
Thank you for pointing out the naming convention. I didn't know how best to name the resources. Concerning the second question, though `abucketlist` and `allbucketlists` are representing the same models, they are each communicating with the model differently. `abucketlist` uses the [GET,POST and PUT] methods while `allbucketlists` uses [GET and POST] method.